public/javascripts/home.js

```diff
@@ -0,0 +1,52 @@
+var home = (function() {
+    var inLibrary = m.prop(currentInLibrary)
+    var playlistName = m.prop()
+    var displayedSongs = m.prop(songs)
+    var initial = 10
+    var langShown = m.prop('All')
+    var langFilter = m.prop([])
+
+    //loadMore function will also add filter for displaed songs in the page
```

Comment:
typo in the comment.
What does it mean that it will also add filter? Should the function name be loadMoreAndApplyFilter in that case?